scripts/benchmarks/run_parakeet.py

```
@@ -0,0 +1,320 @@
+                pairs.append((audio, cand))
+                break
+    return pairs
+
+
+def to_16k_mono(path: Path, tmpdir: Path) -> Path:
+    """Reuse the exact cache directory run_corpus.py writes, so every engine here
+    reads byte-identical audio and no conversion difference can leak into CER."""
+    out = tmpdir / (path.stem + ".16k.wav")
+    if out.exists():
+        return out
+    subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)